notebooks/explore_DL2.ipynb

```diff
@@ -15,6 +15,7 @@
     "from astropy.coordinates.erfa_astrom import ErfaAstromInterpolator, erfa_astrom\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        # Redue precision to save memory:
</pre><p>typo: Reduce</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>